src/services/archiver.service.ts

```diff
@@ -0,0 +1,52 @@
+import archiver, { Archiver } from 'archiver';
+import { Response } from 'express';
+
+import { retrieveLangauge } from "../utils/retrieve-language";
+import { createTempDirectory, removeTempDirectory } from "../utils/temp-dir";
+
+/**
+ * Service providing `archiver` functionality with needed helpers.
```

Comment:
This is a very personal opinion, but I didn't understand what this was about when reading this line. The word `archiver` doesn't help me understanding that what this service does is creating a ZIP file.
After reading the whole class, I did have an idea then. Would it make sense to reword this line to say what the service is really doing?

Reply:
Ok, I will update description. That service is for wrapping `archiver` module to make easier handling of `zip` :)